angular/api-call/README.md

```diff
@@ -0,0 +1,51 @@
+# API Call
+
+## Environment
+
```

Comment:
Could you add more explanation with this one na? I don't understand what is this for.

Reply:
@NorakGithub Already updated kaa